### DISCUSSION TITLE
v3.1.0-beta.x

### DISCUSSION THREAD

**Discussion by johnborges** [capacitor-community/app-icon#21] [Announcements]
### Feature

* Android support

The only method currently supported for Android in this beta is `change()`. You need to specify the icons that need to be disabled as an additional option with the one you want to enable. The android project in [example](https://github.com/capacitor-community/app-icon/tree/android-support/example) is a good place to look for references. Please post general feedback here, and let me know how it works for you.

<hr /><em>This discussion was created from the release <a href='https://github.com/capacitor-community/app-icon/releases/tag/v3.1.0-beta.0'>v3.1.0-beta.0</a>.</em>

**Comment by scr2em** [comment]
@johnborges Thanks for your effort, You are doing a good job. 
When will this be released ?

**Reply by johnborges** [reply]
Do you mean out of beta?

**Reply by scr2em** [reply]
@johnborges yes

**Reply by johnborges** [reply]
There is no set timeline. I'm hoping to gather as much feedback as possible and resolve bugs with the beta first.

**Comment by johnborges** [comment]
[v3.1.0-beta.1](https://github.com/capacitor-community/app-icon/releases/tag/v3.1.0-beta.1) has been published
